database/seeds/RustServiceTableSeeder.php

```diff
@@ -0,0 +1,262 @@
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+use Illuminate\Database\Seeder;
+use Pterodactyl\Models\Service;
+use Pterodactyl\Models\ServiceOption;
+use Pterodactyl\Models\ServiceVariable;
+
+class RustServiceTableSeeder extends Seeder
+{
+    /**
+     * The core service ID.
+     *
+     * @var Models\Service
+     */
+    protected $service;
+
+    /**
+     * Stores all of the option objects.
+     *
+     * @var array
+     */
+    protected $option = [];
+
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $this->addCoreService();
+        $this->addCoreOptions();
+        $this->addVariables();
+    }
+
+    private function addCoreService()
+    {
+        $this->service = Service::updateOrCreate([
+            'author' => config('pterodactyl.service.core'),
+            'folder' => 'rust',
+        ], [
+            'name' => 'Rust',
+            'description' => 'The only aim in Rust is to survive. To do this you will need to overcome struggles such as hunger, thirst and cold. Build a fire. Build a shelter. Kill animals for meat. Protect yourself from other players, and kill them for meat. Create alliances with other players and form a town. Do whatever it takes to survive.',
+            'startup' => './RustDedicated -batchmode +server.port {{SERVER_PORT}} +server.identity "rust" +rcon.port {{RCON_PORT}} +rcon.web true +server.hostname \"{{HOSTNAME}}\" +server.level \"{{LEVEL}}\" +server.description \"{{DESCRIPTION}}\" +server.url \"{{URL}}\" +server.headerimage \"{{SERVER_IMG}}\" +server.worldsize \"{{WORLD_SIZE}}\" +server.seed \"{{SEED}}\" +server.maxplayers {{MAX_PLAYERS}} +rcon.password \"{{RCON_PASS}}\" {{ADDITIONAL_ARGS}}',
+            'index_file' => Service::defaultIndexFile(),
+        ]);
+    }
+
+    private function addCoreOptions()
+    {
+        $script = <<<'EOF'
+apt update
+apt -y --no-install-recommends install curl lib32gcc1 ca-certificates
+
+cd /tmp
+curl -sSL -o steamcmd.tar.gz http://media.steampowered.com/installer/steamcmd_linux.tar.gz
+
+mkdir -p /mnt/server/steam
+tar -xzvf steamcmd.tar.gz -C /mnt/server/steam
+cd /mnt/server/steam
+
+chown -R root:root /mnt
+
+export HOME=/mnt/server
+./steamcmd.sh +login anonymous +force_install_dir /mnt/server +app_update 258550 +quit
+
+mkdir -p /mnt/server/.steam/sdk32
+cp -v linux32/steamclient.so ../.steam/sdk32/steamclient.so
+EOF;
+
+        $this->option['rustvanilla'] = ServiceOption::updateOrCreate([
+            'service_id' => $this->service->id,
+            'tag' => 'rustvanilla',
+        ], [
+            'name' => 'Vanilla',
+            'description' => 'Vanilla Rust server.',
+            'docker_image' => 'tenten8401/pterodactyl-rust',
```

Comment:
I need you to make a branch for Rust from what I can tell. Correct me if I'm wrong on that.